### PR TITLE
Migrate storage pools

### PR DESCRIPTION
**Pull request by howbazaar**
No CI yet, almost there, just storage constraints to go.

(Review request: http://reviews.vapour.ws/r/5472/)

### CODE REVIEW

File: state/migration_import.go [hunk 1]
```diff
@@ -1486,3 +1490,19 @@ func (i *importer) addFilesystemAttachmentOp(fsID string, attachment description
...
+	registry, err := i.st.storageProviderRegistry()
+	if err != nil {
+		return errors.Annotate(err, "getting provider registry")
+	}
+	pm := poolmanager.New(NewStateSettings(i.st), registry)
+
+	for _, pool := range i.model.StoragePools() {
+		_, err := pm.Create(pool.Name(), storage.ProviderType(pool.Provider()), pool.Attributes())
+		if err != nil {
+			return errors.Annotatef(err, "pool %q", pool.Name())
```
Comment (axw) [inline state/migration_import.go]: "creating pool %q" perhaps?

**Comment by axw** [comment]
LGTM

File: state/migration_import.go (hunk 1 above)
Comment (howbazaar) [inline state/migration_import.go]: done

**Comment by howbazaar** [comment]
$$merge$$

**Pull request merged**